jobs.groovy.j2

```diff
@@ -327,7 +327,8 @@ branches.each {
     }
 {# limit execution to jenkins slaves with a particular label                 #}
     label("{{ job_values.on_nodes_with_labels }}")
-    {{ wrappers(job_values, 'none') }}
+{% set repo_dir = "/tmp/workspace/${dashProject}/${dashBranchName}/" + _job_name + "/_trial_temp" %}
+    {{ wrappers(job_values, repo_dir) }}
     {{ scm("${git_url}", "${branchName}") }}
```

Comment:
as we run trial as root, we need to execute the clean the repository of files owned by root before we attempt to git pull